src/components/ModalFooter.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+
+
+export default ({properties})=>{
+
+    const {bicycle, openfire, fee, toilets , dog, drinking_water, reservation}= properties;
+    const availableStyle={
+        width:"10%",
+        border:"1px solid blue",
+        backgroundColor:"rgb(47, 155, 255)"
+    }
+   const unavailableStyle={
+       width:"10%",
+       opacity:"0.5",

```

Comment:
Maybe it is better to increase the opacity for the not selected filter also to be visible a little